test/experimental/new_embed_test.html

```diff
@@ -28,6 +28,7 @@
     <link href="../packages/codemirror/addon/hint/show-hint.css" rel="stylesheet" media="screen">
     <link href="../packages/codemirror/theme/elegant.css" rel="stylesheet" media="screen">
     <script src="../packages/codemirror/codemirror.js" defer></script>
+    <script src="../packages/split/split.min.js" defer></script>
```

Comment:
To make sure this file is accessible when deployed to prod, you'll need to update the regex for packages in `app.yaml`:

https://github.com/dart-lang/dart-pad/blob/master/web/app.yaml

Reply:
cool, fixed in 6e71d20